### PR TITLE
Infinispan 12.1.3.Final and 9.4.23.Final

### PR DESCRIPTION
**Pull request by pruivo**

### CODE REVIEW

File: _data/ispn.yml [hunk 1]
```diff
@@ -193,20 +193,20 @@ old:
...
-      all: https://downloads.jboss.org/infinispan/9.4.20.Final/infinispan-9.4.20.Final-all.zip
-      bin: https://downloads.jboss.org/infinispan/9.4.20.Final/infinispan-9.4.20.Final-minimal.zip
-      server: https://downloads.jboss.org/infinispan/9.4.20.Final/infinispan-server-9.4.20.Final.zip
-      wfmodules: https://downloads.jboss.org/infinispan/9.4.20.Final/infinispan-wildfly-modules-9.4.20.Final.zip
-      docker: jboss/infinispan-server:9.4.20.Final
+      all: https://downloads.jboss.org/infinispan/9.4.23.Final/infinispan-9.4.23.Final-all.zip
+      bin: https://downloads.jboss.org/infinispan/9.4.23.Final/infinispan-9.4.23.Final-minimal.zip
+      server: https://downloads.jboss.org/infinispan/9.4.23.Final/infinispan-server-9.4.23.Final.zip
+      wfmodules: https://downloads.jboss.org/infinispan/9.4.23.Final/infinispan-wildfly-modules-9.4.23.Final.zip
+      docker: jboss/infinispan-server:9.4.23.Final
```
Comment (pruivo) [inline _data/ispn.yml:209]: @ryanemerson are we going to have a new image for 9.4 or should I revert this change?

Comment (ryanemerson) [reply _data/ispn.yml:209]: New images only for `12.1.x` and `13.x`

**Comment by pruivo** [comment]
Added 11.0.11.Final release

**Comment by ryanemerson** [comment]
Thanks @pruivo
